### DISCUSSION TITLE
selectmanybutton scss file

### DISCUSSION THREAD

**Discussion by Just-Dominic** [primefaces/community#1536] [PrimeFaces] [answered]
Hi,

I am looking for the default styling for the selectmanybutton component. I don't see it among the scss files in my folder. Can I download it somewhere? 

Thanks in advance!

**Comment by FlipWarthog** [comment]
Do you mean the SCSS files in this project or another? <https://github.com/primefaces/primefaces-sass-theme>

**Comment by melloware** [comment, marked as answer]
For themes it is here: https://github.com/primefaces/primefaces-sass-theme/blob/main/theme-base/components/button/_button.scss

Core styles before theme is applied are here: https://github.com/primefaces/primefaces/blob/master/primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.css
